src/commands/CommandRouter.ts

```diff
@@ -0,0 +1,150 @@
+import { Client, Events, GuildScheduledEvent, Interaction, Message, REST } from "discord.js";
+import { RESTPostAPIApplicationCommandsJSONBody, Routes } from "discord-api-types/v9";
```

Comment:
Note - rest of API is updated to use v10, not sure if we need to modify payload for this change.